build.rs

```diff
@@ -235,6 +235,13 @@ const ASM_TARGETS: &[AsmTarget] = &[
         asm_extension: "S",
         preassemble: true,
     },
+    AsmTarget {
+        oss: &[UEFI],
```

Comment:
Does it not work correctly if we add UEFI to the `AsmTarget` that has `oss: &[WINDOWS],
        arch: "x86_64"`?